ui/job-view/details/summary/SummaryPanel.jsx

```diff
@@ -56,6 +56,7 @@ class SummaryPanel extends React.PureComponent {
                     icon={faSpinner}
                     pulse
                     className="th-spinner-lg"
+                    aria-label="Loading..."
```

Comment:
These components appear to have `aria-hidden=true` on by default (looking at the devtools inspector) and so the `aria-label` should go on the parent elements these icons are part of (button, div, span, a, etc).